web/src/app/chat/components/input/ActionManagement.tsx

```diff
@@ -335,20 +342,32 @@ function MCPServerItem({
         rounded-lg
         py-2
         mx-1
-        ${isExpanded ? "bg-accent-100 hover:bg-accent-200" : ""}
+        ${isActive ? "bg-accent-100 hover:bg-accent-200" : ""}
+        ${allToolsDisabled ? "opacity-60" : ""}
```

Comment:
Instead of using string-formatting to pass conditional logic to the className, could you please try using the `cn` utility function, please?

```tsx
import { cn } from "@/lib/utils";
```